src/App.jsx

```diff
@@ -14,70 +16,75 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
+  const heading = (
     <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+      {selectedGood
+        ? `${selectedGood} is selected`
+        : 'No goods selected'
+      }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
+      {selectedGood
+        && (
+          <button
+            onClick={() => setSelectedGood('')}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        )}
     </h1>
+  );
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  function handleGoodClick(currentGood) {
+    if (currentGood === selectedGood) {
+      setSelectedGood('');
```

Comment:
```suggestion
      clearGood();
```
Here's an example of code reuse.